packages/sit-onyx/src/styles/mixins/normalize.scss

```diff
@@ -0,0 +1,221 @@
+@mixin apply {
+  html:focus-within & {
+    scroll-behavior: smooth;
+  }
+
+  &,
+  * {
+    &,
+    &::before,
+    &::after {
+      margin: 0;
+
+      box-sizing: border-box;
+
+      -moz-text-size-adjust: none;
+      -webkit-text-size-adjust: none;
+      text-size-adjust: none;
+    }
+
+    &:is(ul, ol) {
+      list-style: none;
```

Comment:
Not sure if this is intended...what if I want to place a simple list in my project inside the app layout?